src/Microsoft.AspNet.Mvc.Core/RedirectToRouteResult.cs

```diff
@@ -77,7 +78,14 @@ public override void ExecuteResult(ActionContext context)
 
         private IUrlHelper GetUrlHelper(ActionContext context)
```

Comment:
To confirm: None of these `GetUrlHelper()` methods are ever called more than once?